frontend-vue/e2e/setup/Dockerfile

```diff
@@ -1,2 +1,2 @@
-FROM softwarepark/contexture:latest
+FROM softwarepark/contexture:0.2.1
```

Comment:
Is this needed because of the flaky backend integration?

Reply:
Yes, but change was not pushed on purpose.